core/ustring.cpp

```diff
@@ -1787,22 +1787,18 @@ int64_t String::hex_to_int64(bool p_with_prefix) const {
 
 int64_t String::bin_to_int64(bool p_with_prefix) const {
 	int len = length();
-	if (len == 0 || (p_with_prefix && len < 3)) {
-		return 0;
-	}
+	ERR_FAIL_COND_V_MSG(p_with_prefix ? len < 3 : len == 0, 0, String("Invalid binary notation length in string ") + (p_with_prefix ? "with" : "without") + "prefix \"" + *this + "\".");
 
 	const CharType *s = ptr();
 
 	int64_t sign = s[0] == '-' ? -1 : 1;
 
 	if (sign < 0) {
-		s++;
+		++s;
```

Comment:
Why change this? It shouldn't make a difference and the dominant style used in the codebase is postfix.